### PR TITLE
Add developer docu to the OpenAPI directory

### PR DESCRIPTION
**Pull request by hennevogel**
What is it and how can you change it?

### CODE REVIEW

File: src/api/public/apidocs-new/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+This is the API specification for the Open Build Service HTTP API.
+
+You can look at this online for the reference server at
+
+https://api.opensuse.org/apidocs/
+
+or at the same location (`/apidocs/`) in your local OBS installation.
+
+We are using the [OpenAPI Specification (OAS) version 3.0.0](https://spec.openapis.org/oas/v3.0.0)
+to document the API and and the [Swagger UI](https://swagger.io/tools/swagger-ui/) tool to display
```
Comment (saraycp) [inline src/api/public/apidocs-new/README.md:12]:
```suggestion
to document the API and the [Swagger UI](https://swagger.io/tools/swagger-ui/) tool to display
```

File: src/api/public/apidocs-new/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+it.
+
+## How to contribute?
+
+Are you missing documentation for a endpoint? Did you find errors or oversights
```
Comment (saraycp) [inline src/api/public/apidocs-new/README.md:17]:
```suggestion
Are you missing documentation for an endpoint? Did you find errors or oversights
```

File: src/api/public/apidocs-new/README.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+in the current documentation? Check out the [OpenAPI Guide](https://swagger.io/docs/specification/about/), our [OBS contribution guide](https://github.com/openSUSE/open-build-service/blob/master/CONTRIBUTING.md) and contribute fixes please!
```
Comment (saraycp) [inline src/api/public/apidocs-new/README.md:18]:
```suggestion
in the current documentation? Check out the [OpenAPI Guide](https://swagger.io/docs/specification/about/), our [OBS contribution guide](https://github.com/openSUSE/open-build-service/blob/master/CONTRIBUTING.md) and contribute fixes, please!
```

**Review by saraycp — CHANGES_REQUESTED** [review]

File: src/api/public/apidocs-new/README.md (hunk 3 above)
Comment (hennevogel) [reply src/api/public/apidocs-new/README.md]: pushed

**Review by bmwiedemann — APPROVED** [review]

**Review by saraycp — CHANGES_REQUESTED** [review]
Please rebase on master. Since PR #14438 is merged, the apidocs-new directory no longer exists. The path to the README file is wrong.

**Comment by hennevogel** [comment]
> Please rebase on master

done

**Pull request merged**